packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -96,6 +101,21 @@ export interface OdspErrorResponse {
     };
 }
 
+/** Redirect error from ODSP  */
+export class OdspRedirectError extends LoggingError {
+    readonly errorType = DriverErrorType.fileNotFoundOrAccessDeniedError;
```

Comment:
Do we have a UT that ensures that 404's with redirectLocation in the error response get this error type?

Reply:
yepyep

Reply:
there was a test in epochtracker, and the type was locationRedirection